pages/media.jsx

```diff
@@ -54,7 +54,7 @@ const PAST_COVERAGE = [
   {
     media: `t3n`,
     title: `Wie steht es um die Gesundheit des Internets? Auf Spurensuche beim Mozilla-Festival`,
-    link: `http://t3n.de/news/mozilla-festival-internet-gesundheit-761360/3/`,
+    link: `https://web.archive.org/web/20170112212535/http://t3n.de/news/mozilla-festival-internet-gesundheit-761360/3/`,
```

Comment:
Hey @KevZawacki the link for `tn3` on https://mozillafestival.org/media is not available anymore. Is it okay if we fix the link to `https://web.archive.org/web/20170112212535/http://t3n.de/news/mozilla-festival-internet-gesundheit-761360/3/` or you have other suggestion?

Reply:
pinging @KevZawacki again in case you missed this